package.json

```diff
@@ -2,36 +2,27 @@
   "name": "klicker-knight",
   "version": "0.0.0",
   "scripts": {
-    "clean": "rimraf lib",
     "clean:game": "rm -r saves/*.json",
     "compile:gameStateSchema": "json2ts src/utils/types/gameState.schema.json > src/utils/types/gameState.d.ts",
-    "dev": "node bin/run",
+    "klicker-knight": "ts-node src/klickerKnight",
     "lint:md": "markdownlint **/*.md --ignore node_modules",
     "lint:ts": "eslint --ext .ts,.js .",
     "lint:yaml": "yamllint --ignore=node_modules/**/*.yaml **/*.yaml",
-    "prepack": "npm run clean && tsc -b",
-    "test": "nyc --extension .ts mocha \"tests/**/*.test.ts\"",
-    "typecheck": "tsc --noEmit -P tests/tsconfig.json ",
-    "update-readme": "oclif-dev readme"
-  },
-  "bin": {
-    "klicker-knight": "./bin/run"
+    "test:e2e": "mocha \"tests/e2e/**/*.test.ts\"",
+    "test:src": "nyc --extension .ts mocha \"tests/src/**/*.test.ts\"",
```

Comment:
Should we start calling these unit tests?

Reply:
The databaseUtil test is technically an integration test. I didn't want to split out unit/integration tests and I thought `unit` might be confusing.